includes/class-newspack-popups.php

```diff
@@ -14,6 +14,8 @@ final class Newspack_Popups {
 
 	const NEWSPACK_PLUGINS_CPT = 'newspack_popups_cpt';
 
+	const NEWSPACK_POPUP_PREVIEW_QUERY_PARAM = 'newspack_popup_preview_id';
```

Comment:
I tend to prefix plugin options with the full slug of the plugin, so this would be `newspack_popups_preview_id`.